web/apps/labelstudio/tsconfig.app.json

```diff
@@ -19,5 +19,5 @@
     "src/**/*.spec.jsx",
     "src/**/*.test.jsx"
   ],
-  "include": ["src/**/*.js", "src/**/*.jsx", "src/**/*.ts", "src/**/*.tsx"]
+  "include": ["src/**/*.js", "src/**/*.jsx", "src/**/*.ts", "src/**/*.tsx", "../../libs/ui/src/hooks/useOrgValidation.ts"]
```

Comment:
This hook is not a UI concern, if we want to have some common package for shared logic that sits at the Application level we would need to create a separate package for this.